helpers/db_indexes.go

```diff
@@ -0,0 +1,20 @@
+package helpers
+
+import (
+	"context"
+	"time"
+
+	"github.com/drift-org/backend/models"
+)
+
+/*
+Setup the indexes of all the collections
+
+As we create indexes for models, make sure to add them to this function
+*/
+func SetupIndexes() {
+	ctx, canc := context.WithTimeout(context.Background(), 5*time.Second)
```

Comment:
Add documentation for why this is necessary.

Reply:
I need a context in order to start creating indexes

Reply:
Ok, but if you just needed a context, you have `context`. So why specifcally do you need `context.WithTimeout(context.Background(), 5*time.Second)` (specifcally 5 seconds)?

Reply:
This is just so it doesn't run forever and the channel will closed and context will be canceled when time is up.

Reply:
That sounds great. I would add a comment that explains that, and also make 5 a constant (something like`INDEX_CREATION_TIME_THRESHOLD` would work).